core/shared/platform/include/platform_api_vmcore.h

```diff
@@ -65,6 +65,12 @@ os_vprintf(const char *format, va_list ap);
  */
 uint64
 os_time_get_boot_microsecond(void);
+/* Get current clock time*/
```

Comment:
```suggestion

/* Get current clock time*/
```
also, please add extra empty line between previous function and this comment.